rust/Cargo.toml

```diff
@@ -1,44 +1,44 @@
 [package]
-name = "ics23"
-version = "0.8.1"
 authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
+description = "Merkle proof verification library - implements Cosmos ICS23 Spec"
 edition = "2021"
 exclude = ["codegen", "no-std-check"]
-description = "Merkle proof verification library - implements Cosmos ICS23 Spec"
-repository = "https://github.com/confio/ics23/tree/master/rust"
 license = "Apache-2.0"
+name = "ics23"
+repository = "https://github.com/cosmos/ics23/tree/master/rust"
```

Comment:
before merge I need to read on how this effects cargo

Reply:
No effect aside from updating the crate's page on crates.io